ipn/backend.go

```diff
@@ -156,4 +157,7 @@ type Backend interface {
 	// make sure they react properly with keys that are going to
 	// expire.
 	FakeExpireAfter(x time.Duration)
+	// Ping attempts to start connecting to the given IP and sends a Notify
```

Comment:
Nit: "attempts to start" feels duplicative. Would "attempts to" or "starts" be enough by themselves?